remediation/fence-agents-remediation/fence-agents-remediation.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Fence Agents Remediation
+nav_order: 3
+has_children: true
+parent: Remediation
+---
+
+## Fence Agents Remediation
+
+Generally Available
+{: .label .label-green }
+
+[Fence Agents Remediation](https://github.com/medik8s/fence-agents-remediation#readme) (FAR) is a Kubernetes operator that uses external tools to remediate unhealthy nodes. The tools are a set of well-known [upstream fence agents (FA)](https://github.com/ClusterLabs/fence-agents), where each one of them can be used for different environments, to fence a node using a reboot action. Moreover, FAR can be used by the [Node HealthCheck Operator](https://github.com/medik8s/node-healthcheck-operator#readme) (or any other operator that supports the external remediation API) as an external remediator.
```

Comment:
I think the terminology here is confusing, It looks like the reboot done by the FA is synonymous to remediation.
I think this terminology is wrong, the reboot IMO would be the fencing where the remediation is the complete process which includes the workload deletion and Node recovery.